{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "1cb3179a-ab08-47c9-a549-a7b4968b3721-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "b31df4ff-41c9-4a43-9966-a818953dfa22-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "fd769f1f-6798-40c8-98fe-75b0f11d732f-attachment.txt", "type": "text/plain"}], "start": 1632453998364, "stop": 1632454000504, "uuid": "f69c0059-5b71-42e2-a673-87b82a71a10a", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "20812-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}